SU-SWS/stanford_basic
# Update Decanter to v 6.1.1 and compile.

# READY FOR REVIEW 

# Summary
- Updates Decanter to version 6.1.1

# Review By (Date)
- Sept 4th

# Urgency
- High

# Steps to Test

1. Review code
2. Check out this branch on your local build
3. Review the alerts to validate new styles are there.
